### PR TITLE
Test drone

### PR DESCRIPTION
**Pull request by MorrisJobke**

### CODE REVIEW

**Comment by MorrisJobke** [comment]
test

**Comment by MorrisJobke** [comment]
Some tests doesn't seem to be executed. Maybe a space too much in the `when` statement for the step?

File: .drone.yml [hunk 1]
```diff
@@ -1,79 +1,135 @@
...
+    - DB: NODB
+      PHP: 5.6
+    - DB: NODB
+      PHP: 7.01
+    - DB: NODB
+      PHP: 7.1
+    - DB: sqlite
+      PHP: 5.6
+    - DB: sqlite
+      PHP: 7.01
```
Comment (MorrisJobke) [inline .drone.yml]: This is a workaround. It seems that there is an issue with parsing and matrix and when comparision fails if the value ends on a `0`.

File: .drone.yml [hunk 2]
```diff
@@ -1,79 +1,135 @@
...
+    - DB: sqlite
+      PHP: 7.1
+    - DB: mysql
+      PHP: 5.6
```
Comment (MorrisJobke) [inline .drone.yml]: Downside here: there are 5 minutes without output which causes drone to kill it.

File: .drone.yml [hunk 3]
```diff
@@ -1,79 +1,135 @@
...
+    - DB: postgres
+      PHP: 5.6
```
Comment (MorrisJobke) [inline .drone.yml]: Downside here: there are 5 minutes without output which causes drone to kill it.

Comment (MorrisJobke) [inline .drone.yml]: Maybe better on the more powerful CI server instead of on my own slow server.

**Comment by MorrisJobke** [comment]
cc @rullzer for the current status

File: .drone.yml (hunk 3 above)
Comment (rullzer) [inline .drone.yml]: do we really have tests that go 5 min without output?

Comment (MorrisJobke) [inline .drone.yml]: Now only the clone is failing. I raised the timeout to 60 minutes. let's see :)

**Comment by rullzer** [comment]
I must say I like this a lot better already!

Maybe not for now but I'm writing it down here now

- [ ] Simplify matrix. So we build everything for everything. 
  * [ ] PHPVersions: 5.6, 7.0, 7.1
  * [ ] DB: NODB, sqlite, mysql, postgres (oracle?)

Need to figure out how js comes into play here.
But now that it can run faster I'd like to run it all basically ;)

**Comment by MorrisJobke** [comment]
> But now that it can run faster I'd like to run it all basically ;)

First of all the external storage and smashbox tests should be added ;)

**Comment by rullzer** [comment]
Good point!

**Pull request closed without merging**